docs/alphafeatures.md

```diff
@@ -423,7 +423,7 @@ The following section will give the steps to take snapshot and clone the cStor v
    ```
 3. Get the YAML for snapshot creation of a PVC using the following command:
    ```
-   wget https://raw.githubusercontent.com/openebs/cstor-csi/master/deploy/snapshot.yaml
+   wget https://github.com/openebs/cstor-csi/blob/v1.9.x/examples/csi-snapshot.yaml
    ```
    In this example, downloaded file is saved as `snapshot.yaml`.
 4. Edit the snapshot.yaml which is created in previous step to update:
```

Comment:
Edit the snapshot.yaml which is created in previous step to update: -> Edit the `csi-snapshot.yaml` which is created in the previous step to update: